lms/djangoapps/support/static/support/js/certificates_factory.js

```diff
@@ -0,0 +1,11 @@
+;(function (define) {
```

Comment:
@AlasdairSwan These are the files I need you to review.  Hooray for RequireJS optimizer in the LMS!